lib/matplotlib/mlab.py

```diff
@@ -2867,14 +2867,19 @@ def get_func(name, item, func):
         'print':  'print_',
         }
 
-    def get_converters(reader):
+    def get_converters(reader, comments):
```

Comment:
need to make `comments` optional. Perhaps defaults to an empty tuple (which is non-mutable, so it is safe). Should probably also have a line that coerces `comments` into a tuple since `startswith()` does not accept a list.